docs/HISTORY.txt

```diff
@@ -5,10 +5,8 @@ Changelog
 1.10.3 (unreleased)
 -------------------
 
-- Freezer: disable freezing while committing to database for preventing pickling errors.
-  [jone]
-
-- Freezer bugfix: replace datetime instances when leaving freeze context manager.
+- Freezer: reimplement "now" patching with forbiddenfruit.
+  This fixes problems with pickling and comparison of frozen datetime objects.
```

Comment:
This changelog got messed up a bit I think - `1.10.3` has been released in the meantime (containing "disable while committing to db" and "... when leaving context manager"), but this entry replaced the other two retroactively in the existing `1.10.3` release notes.

@jone Could you take a look at that and fix it? (My suggestion would be to simply re-add the old entries to 1.10.3, and move the new entry to a 1.11.0 section)